packages/react/src/menu/root/MenuRoot.test.tsx

```diff
@@ -761,7 +765,7 @@ describe('<Menu.Root />', () => {
             <style dangerouslySetInnerHTML={{ __html: style }} />
             <button onClick={() => setOpen(false)}>Close</button>
             <Menu.Root open={open}>
-              <Menu.Positioner>
+              <Menu.Positioner keepMounted>
```

Comment:
This now tests a different case. Does it not work without `keepMounted` anymore?

Reply:
Nope. It unmounts before it gets a chance to fire

Reply:
`queryByRole` tests for visibility though, which is why the test still passes. It sees `hidden` and thinks it's unmounted and will return `null`. So it tests almost the same thing still.